pkg/storaged/storage-controls.jsx

```diff
@@ -140,11 +140,11 @@ export const StorageBlockNavLink = ({ client, block }) => {
 
     const link = (
         <Button isInline variant="link" onClick={() => { cockpit.location.go(parts.location) }}>
-            {parts.link}
+            {cockpit.format(parts.format, parts.link)}
```

Comment:
Split out and explain.

Reply:
This has nothing to do with Stratis, let's just remove it completely. No idea anymore why I did this...

Reply:
Done.